az-core/src/main/java/azkaban/Constants.java

```diff
@@ -723,6 +723,12 @@ public static class ContainerizedDispatchManagerProperties {
     public static final String KUBERNETES_WATCH_ENABLED = KUBERNETES_WATCH_PREFIX + "enabled";
     public static final String KUBERNETES_WATCH_EVENT_CACHE_MAX_ENTRIES =
         KUBERNETES_WATCH_PREFIX + "cache.max.entries";
+    public static final String KUBERNETES_WATCH_CALL_TIMEOUT = KUBERNETES_WATCH_PREFIX + "call"
```

Comment:
nitpick : Please put the suffixes in one line. It adds to code readability and search. Applies to line 731 as well.

Reply:
+1